components/EditForm/index.js

```diff
@@ -1,18 +1,16 @@
-import {
-  StyLabel,
-  StyInput,
-  StySliderLabel,
-  StySelect,
-  StyDiv,
-  StySlider,
-  StyInputTwo,
-  StyLabelTwo,
-  StyButton,
-} from "../Form";
-
 import { useState } from "react";
-import styled from "styled-components";
-
+import {
+  StyledForm,
+  StyledLabel,
+  StyledInput,
+  StyledSliderLabel,
+  StyledSlider,
+  StyledSelect,
+  StyledLabelVariant,
+  StyledInputVariant,
+  StyledButton,
+  StyledDivGridVariant,
+} from "@/public/lib/styled-components";
```

Comment:
I'm genuinely curious how the styled components ended up in the public folder. 🤔